Plugins/Creature/NWScript/nwnx_creature.nss

```diff
@@ -1092,44 +1120,22 @@ void NWNX_Creature_SetAlignmentLawChaos(object creature, int value)
 
 int NWNX_Creature_GetClericDomain(object creature, int index)
 {
-    string sFunc = "GetClericDomain";
-
-    NWNX_PushArgumentInt(NWNX_Creature, sFunc, index);
-    NWNX_PushArgumentObject(NWNX_Creature, sFunc, creature);
-
-    NWNX_CallFunction(NWNX_Creature, sFunc);
-    return NWNX_GetReturnValueInt(NWNX_Creature, sFunc);
+    return NWNX_Creature_GetDomain(creature, CLASS_TYPE_CLERIC, index);
 }
 
 void NWNX_Creature_SetClericDomain(object creature, int index, int domain)
 {
-    string sFunc = "SetClericDomain";
-
-    NWNX_PushArgumentInt(NWNX_Creature, sFunc, domain);
-    NWNX_PushArgumentInt(NWNX_Creature, sFunc, index);
-    NWNX_PushArgumentObject(NWNX_Creature, sFunc, creature);
-
-    NWNX_CallFunction(NWNX_Creature, sFunc);
+    return NWNX_Creature_SetDomain(creature, CLASS_TYPE_CLERIC, index, domain);
```

Comment:
Don't need a `return` here and could you add a `WriteTimestampedLogEntry("WARNING: NWNX_Creature_SetClericDomain() is deprecated, please use NWNX_Creature_SetDomain()");`?